css/main.css

```diff
@@ -27,10 +27,49 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+#desktop-nav,
+#desktop-nav ul {
   display: none;
 }
 
+#mobile-menu-container {
+  display: none;
+  background-color: #fff;
```

Comment:
- [x] I kindly suggest you to follow the project template! Even thought creativity is a good thing to have :+1:  The project requirements specify that the mobile menu should looks like this:

![image](https://user-images.githubusercontent.com/28903404/139599240-b2537a67-8b50-4216-bd7d-bd194351f74b.png)
The first step would be to change the background color!
```suggestion
  background-color: "Correct template color here";
```

Reply:
Thanks, I got it!